content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,28 @@
+# Getting Started using Entity Framework Core
+by [The Raikes School](http://cse.unl.edu/raikes-school)
+
+## What is Entity Framework Core?
```

Comment:
What is Entity Framework Core? has been removed. Consider renaming this section.